damnit/backend/extract_data.py

```diff
@@ -204,6 +204,10 @@ def extract_and_ingest(self, proposal, run, cluster=False,
 
             python_cmd = [sys.executable, '-m', 'damnit.backend.extract_data',
                           '--cluster-job', str(proposal), str(run), run_data.value]
+            if len(match) > 0:
+                for m in match:
```

Comment:
should you also check that any of the match is actually a cluster variable and only start the job in this case?

Reply:
That's already done a couple lines above:
```python
        # Launch a Slurm job if there are any 'cluster' variables to evaluate
        ctx =       self.ctx_whole.filter(run_data=run_data, name_matches=match, cluster=cluster)
        ctx_slurm = self.ctx_whole.filter(run_data=run_data, name_matches=match, cluster=True)
```

This change is only to forward those match arguments to the slurm job.

Reply:
I see, thanks for the clarification. LGTM